packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -42,14 +43,15 @@ interface Props extends Themeable {
   timeZone: TimeZone;
   allowDetails?: boolean;
   logsSortOrder?: LogsSortOrder | null;
+  escapedNewlines?: boolean;
```

Comment:
what about `forceEscape` instead?
or something else in an "active" form, like `showEscaped`, would it make it clearer?